src/mixins/typography/mixins.scss

```diff
@@ -5,17 +5,25 @@
 
 $typography: null !default;
 
-@mixin responsive-font-emit($tier-map) {
+@mixin responsive-font-emit($tier-map, $important: false) {
   @if meta.type-of($tier-map) == 'map' {
     $size: map.get($tier-map, 'size');
     $lh: map.get($tier-map, 'line-height');
 
     @if $size {
-      font-size: $size;
+      @if $important {
```

Comment:
oh wow, why do we need to handle !important, that seems to be the case for an exception that might fall out of the usage of this mxin, it's already pretty tricky to deal with this sass, why adding complexity to it..?